internal/tools/aws/database.go

```diff
@@ -41,11 +46,45 @@ func (d *RDSDatabase) Provision(store model.InstallationDatabaseStoreInterface,
 
 // Teardown removes all AWS resources related to a RDS database.
 func (d *RDSDatabase) Teardown(keepData bool, logger log.FieldLogger) error {
-	err := rdsDatabaseTeardown(d.installationID, keepData, logger)
+	logger.Info("Tearing down AWS RDS database")
+	awsID := CloudID(d.installationID)
+
+	err := d.client.secretsManagerEnsureRDSSecretDeleted(awsID, logger)
+	if err != nil {
+		return errors.Wrap(err, "unable to delete RDS secret")
+	}
+
+	if !keepData {
+		err = d.client.rdsEnsureDBClusterDeleted(awsID, logger)
+		if err != nil {
+			return errors.Wrap(err, "unable to delete RDS DB cluster")
+		}
+		logger.WithField("db-cluster-name", awsID).Debug("AWS RDS DB cluster deleted")
```

Comment:
return after this log message and eliminate `else`

Reply:
Or flip the conditional and short-circuit (`if keepData { log; return nil}`)  to eliminate the `else`, up to you.

Reply:
This code was there before but I will change it anyway.